templates/voting_platform/finish.html.twig

```diff
@@ -18,23 +18,38 @@
+                    <path fill="#C59C6D" fill-rule="evenodd"
+                          d="M731.947374,1393.66179 L950,1570 L950,1570 L490,1570 L708.052626,1393.66179 C715.020828,1388.02663 724.979172,1388.02663 731.947374,1393.66179 Z"
+                          transform="translate(-490 -1389)"/>
                 </svg>
 
-                <svg class="body" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 460 410">
-                  <defs>
-                    <rect id="body-a" width="459.002" height="309.003" x="0" y="0"/>
-                  </defs>
-                  <g fill="none" fill-rule="evenodd" transform="translate(0 100)">
-                    <use fill="#C19664" transform="translate(0 .997)" xlink:href="#body-a"/>
-                    <rect class="paper" width="415.098" height="278.103" fill="#FFF" transform="translate(21.952 17)"/>
-                    <polygon fill="#EDC699" points="0 .498 230.499 155 0 309.502"/>
-                    <polygon fill="#EDC699" points="228.503 .498 459.002 155 228.503 309.502" transform="rotate(-180 343.753 155)"/>
-                    <path fill="#D7AC7B" d="M137.299578,-12.2015069 L314.578908,207.243924 C319.317445,213.109529 319.317455,221.487465 314.578931,227.353081 L137.299578,446.799578 L137.299578,446.799578 L137.299578,-12.2015069 Z" transform="rotate(-90 230 217.299)"/>
-                  </g>
+                <svg class="body" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
+                     viewBox="0 0 460 410">
+                    <defs>
+                        <rect id="body-a" width="459.002" height="309.003" x="0" y="0"/>
+                    </defs>
+                    <g fill="none" fill-rule="evenodd" transform="translate(0 100)">
+                        <use fill="#C19664" transform="translate(0 .997)" xlink:href="#body-a"/>
+                        <rect class="paper" width="415.098" height="278.103" fill="#FFF"
+                              transform="translate(21.952 17)"/>
+                        <polygon fill="#EDC699" points="0 .498 230.499 155 0 309.502"/>
+                        <polygon fill="#EDC699" points="228.503 .498 459.002 155 228.503 309.502"
+                                 transform="rotate(-180 343.753 155)"/>
+                        <path fill="#D7AC7B"
+                              d="M137.299578,-12.2015069 L314.578908,207.243924 C319.317445,213.109529 319.317455,221.487465 314.578931,227.353081 L137.299578,446.799578 L137.299578,446.799578 L137.299578,-12.2015069 Z"
+                              transform="rotate(-90 230 217.299)"/>
+                    </g>
                 </svg>
             </div>
         </div>
     </div>
 {% endblock %}
+
+{% block final_javascripts %}
+    {{ parent() }}
+
+    <script type="text/javascript">
+        Kernel.onLoad(function () { addClass(dom('.envelope'), 'active'); });
```

Comment:
if there no condition to add the class, why not adding it directly in twig ?

Reply:
oh yes, it's for starting CSS animation only after the page is loaded.